examples/ale_regression_boston.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>It's usually helpful to print the feature names so that the user has a clue what are the independent variables if they don't run the nb and just look at the example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>